addons/rating/views/rating_templates.xml

```diff
@@ -16,6 +16,16 @@
             </t>
         </template>
 
+        <!-- External page: Invalid partner rating -->
+        <template id="invalid_partner_rating_external_page_view" primary="True"
```

Comment:
Can we rename to "rating_external_page_invalid_partner" to match the naming of others?